pages/public_cloud/containers_orchestration/managed_kubernetes/expose_your_services_using_a_load_balancer/guide.en-gb.md

```diff
@@ -0,0 +1,399 @@
+
+  This annotation is the ID of a subnet belonging to the floating network, if specified, it takes precedence over `loadbalancer.openstack.org/floating-subnet` or `loadbalancer.openstack.org/floating-tag`.
+
+- `loadbalancer.openstack.org/floating-subnet-tags` //NOT SUPPORTED on OVH
+
+  This annotation is the tag of a subnet belonging to the floating network.
+
+- `loadbalancer.openstack.org/class` //NOT SUPPORTED on OVH
+
+  The name of a preconfigured class in the config file. If provided, this config options included in the class section take precedence over the annotations of floating-subnet-id, floating-network-id, network-id, subnet-id and member-subnet-id . See the section below for how it works.
+
+
+- `loadbalancer.openstack.org/default-tls-container-ref` // NOT SUPPORTED
+
+  Reference to a tls container. This option works with Octavia, when this option is set then the cloud provider will create an Octavia Listener of type `TERMINATED_HTTPS` for a TLS Terminated loadbalancer.
+  Format for tls container ref: `https://{keymanager_host}/v1/containers/{uuid}`
+
+  When `container-store` parameter is set to `external` format for `default-tls-container-ref` could be any string.
+
+### Features
+#### Resize your LoadBalancer //TODO (sample on github)
+There is no proper way to 'resize' your loadbalancer yet (work in progress). The best alternative to change the flavor, is to recreate a new Kubernetes Service that will use the same public IP than the previous one.
+You can find the HowTo and example on our public Github: https://github.com/ovh/public-cloud-examples
+
+
+- First, make sure that the existing service is using the `loadbalancer.openstack.org/keep-floatingip` annotation, if not the public floating IP will be released. (It can be added after the service creation).
+- Get the public IP of your existing service
+```shell
+$ kubectl get service my-small-lb
+NAME                 TYPE           CLUSTER-IP    EXTERNAL-IP      PORT(S)        AGE
+test-lb-todel        LoadBalancer   10.3.107.18   141.94.215.240   80:30172/TCP   12m
+```
+- Create a new service  with the new expected flavor:
+  ```yaml
+  apiVersion: v1
+  kind: Service
+  metadata:
+    name: my-medium-lb
+    annotations:
+      loadbalancer.ovhcloud.com/class: "octavia"
+      loadbalancer.ovhcloud.com/flavor: "medium"
+    labels:
+      app: demo-upgrade
+  spec:
+    loadBalancerIP: 141.94.215.240 # Use the IP address from the previous service
+    ports:
+    - name: client
+      port: 80
+      protocol: TCP
+      targetPort: 80
+    selector:
+      app: nginx
+    type: LoadBalancer
+    ```
+- Until the deletion of the previous service, this Service will only deploy the LoadBalancer without a floating IP.
+- When the Floating IP is available (the deletion of the initial LB service will unbound the IP), the floating ip will be attach to this new LB.
+
+> [!warning]
+>
+> Changing the flavor will create some outage.
```

Comment:
C'est a dire ?

Reply:
Comme indiqué au dessus on delete/create le LB donc le temps de la bascule il y a une intéruption

Reply:
Le terme "some outage" peut indiqué que l'utilisateur aura une panne, un probleme, cela a une connotation negative.
Plutot que d'utiliser le terme "some outage", dire : "Changing the flovor will create a new LB and delete the existing one so it might take some time"

un wording du genre ?